doc/Command_Index_internal.tex

```diff
@@ -430,6 +430,42 @@ \subsection{Note Alignment Type}\label{notesalign}
   12 & divisio finalis
  \end{tabular}
 
+\subsection{Episemus Special}\label{EpisemusSpecial}
```

Comment:
I'll admit I was just copying what I had done before and hadn't looked at the code to see if the meaning of the arguments had changed.